.github/workflows/go.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-22.04
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      
+    # not sure if this part is necessary for the current repo
+    - name: Get protoc
+      run: |
+        sudo curl -L https://github.com/protocolbuffers/protobuf/releases/download/v3.15.0/protoc-3.15.0-linux-x86_64.zip > /tmp/protoc.zip
+        sudo unzip /tmp/protoc.zip -d /usr/local
+        sudo chown -R $USER /usr/local
+        
+    - name: Get dependencies
+      run: go get -v -t -d ./...
+             
+    - name: Build
+      run: make all
```

Comment:
It looks like there are CI failures - I think we either need to add a Makefile or change this to use `go build`.

Reply:
created https://github.com/google/gnostic-models/pull/6